wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureGuestLinkViewController.swift

```diff
@@ -399,6 +400,21 @@ extension CreateSecureGuestLinkViewController: UITextFieldDelegate {
 
     @objc
     func textFieldDidChange(_ textField: UITextField) {
+        evaluateTextfieldsAndToggleCreateLinkButtonState()
+    }
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if textField == securedGuestLinkPasswordTextfield {
```

Comment:
We can test for the same pointer:
```suggestion
        if textField === securedGuestLinkPasswordTextfield {
```

Reply:
Done!